src/main/resources/com/google/api/codegen/csharp/wrapper.snip

```diff
@@ -190,13 +190,13 @@
             /// </remarks>
             public CallSettings {@method.grpcName}Settings { get; set; } = new CallSettings
             {
-                RetrySettings = new RetrySettings
+                Timing = CallTiming.FromRetry(new RetrySettings
```

Comment:
Do you mean then the retry configuration in the yaml has no response statuses that allow retry?
I *think* the idea is that even in this case we should still use the retry configuration, as it will have a suitable initial timeout. I'll check what other languages are doing.